extensions/common/iam/identity-trust/identity-trust-service/src/main/java/org/eclipse/edc/iam/identitytrust/service/IdentityAndTrustService.java

```diff
@@ -133,6 +133,15 @@ public Result<ClaimToken> verifyJwtToken(TokenRepresentation tokenRepresentation
         var claimToken = claimTokenResult.getContent();
         var accessToken = claimToken.getStringClaim(PRESENTATION_TOKEN_CLAIM);
         var issuer = claimToken.getStringClaim(ISSUER);
+        var scopeResult = extractScope(accessToken);
```

Comment:
I'm afraid we can't do this. The shape of the `accessToken` is not defined in the spec, so it could be any string. It might not contain a `scope` claim, in fact, it might not even be a JWT. From the [spec](https://eclipse-dataspace-dcp.github.io/decentralized-claims-protocol/HEAD/#verifiable-presentation-access-token):

> The format of the token is implementation-specific and therefore should be treated as an opaque string by the Verifier.